simulation/modules/plotpoint.py

```diff
@@ -0,0 +1,72 @@
+import attr
+import matplotlib.pyplot as plt
+import numpy as np
+
+from simulation.module import Module, ModuleState
+from simulation.state import State
+
+
+@attr.s(kw_only=True)
+class PlotPointState(ModuleState):
+    step_num: int = attr.ib(default=0)
+    fungal_burdens: np.ndarray = attr.ib(factory=lambda: np.zeros(1000, dtype=int))
+    macrophage_counts: np.ndarray = attr.ib(factory=lambda: np.zeros(1000, dtype=int))
+    neutrophil_counts: np.ndarray = attr.ib(factory=lambda: np.zeros(1000, dtype=int))
+    time_steps: np.ndarray = attr.ib(factory=lambda: np.zeros(1000, dtype=float))
+
+
+class PlotPoint(Module):
```

Comment:
Is there a reason you choose this name? What's the "point"?

Reply:
Oh the name was from before we changed the module to have the arrays